src/event-listeners/index.js

```diff
@@ -77,6 +83,56 @@ export default function setupEventListeners(store, canvas) {
           // prevent the browser from navigating back to the last page
           event.preventDefault()
         }
+      case 'ControlLeft':
+        store.dispatch(setCtrlKeyDown())
+        break
+      case 'KeyC':
+        if (state.ui.keyboard.ctrlKeyDown) {
+          if (state.ui.selection.selectedGoals.length) {
+            // use first
+            store.dispatch(setGoalClone(state.ui.selection.selectedGoals))
+          }
+        }
+        break
+      case 'KeyV':
+        if (state.ui.keyboard.ctrlKeyDown) {
+          if (state.ui.goalClone.goals.length) {
+            state.ui.goalClone.goals.forEach(value => {
```

Comment:
I think this part should be refactored... 
create a new file somewhere, that exports a function that accepts the necessary arguments and performs these actions... 
e.g. 
`cloneGoals.js`
```javascript

export default function cloneGoals(store, goalsToClone, goalMembers) {
  ...
}
```

Reply:
re: questions 3 and 4 : no cloning of priority votes (yet) and no copying of edges. If using command + D, the new card should appear the closest possible to the original card.